pkg/start/start.go

```diff
@@ -542,3 +551,26 @@ func (o *Options) NewControllerContext(cb *ClientBuilder, startingFeatureSet str
 	}
 	return ctx, nil
 }
+
+// parseAlwaysEnableCapabilities parses the string list of capabilities
+// into two lists of configv1.ClusterVersionCapability: known and unknown.
+func parseAlwaysEnableCapabilities(caps []string) ([]configv1.ClusterVersionCapability, []configv1.ClusterVersionCapability) {
+	var (
+		knownCaps   []configv1.ClusterVersionCapability
+		unknownCaps []configv1.ClusterVersionCapability
+	)
+	for _, c := range caps {
+		known := false
+		for _, kc := range configv1.KnownClusterVersionCapabilities {
```

Comment:
Is `KnownClusterVersionCapabilites` a map?  If so, you don't need to use range to check every entry.

Reply:
No, it's [a slice](https://github.com/openshift/cluster-version-operator/blob/6a76ba95ed441893e1bdf6616c47701c0464b7f4/vendor/github.com/openshift/api/config/v1/types_cluster_version.go#L401).